lib/services/service-scaffold.ts

```diff
@@ -92,7 +92,7 @@ export abstract class ServiceScaffold<T> extends WorkerClient<T> implements Serv
 			// This passes to the inheriting class responsibility for understanding the payload structure.
 			this.expressApp.use(bodyParser.text({type: '*/*'}));
```

Comment:
Related to comment in the Github Service, I'd far rather a set of content types and parsers to use were passed in, rather than just `any`.